packages/edit-site/src/components/sidebar-navigation-screen-global-styles/index.js

```diff
@@ -69,15 +77,67 @@ function SidebarNavigationScreenGlobalStylesContent() {
 	// rendering the iframe. Without this, the iframe previews will not render
 	// in mobile viewport sizes, where the editor canvas is hidden.
 	return (
-		<BlockEditorProvider
-			settings={ storedSettings }
-			onChange={ noop }
-			onInput={ noop }
-		>
-			<div className="edit-site-sidebar-navigation-screen-global-styles__content">
+		<>
+			<BlockEditorProvider
+				settings={ storedSettings }
+				onChange={ noop }
+				onInput={ noop }
+			>
 				<StyleVariationsContainer />
-			</div>
-		</BlockEditorProvider>
+			</BlockEditorProvider>
+		</>
+	);
+}
+
+function SidebarNavigationScreenGlobalStylesFooter( { onClickRevisions } ) {
+	const { revisions, isLoading } = useGlobalStylesRevisions();
+	const { revisionsCount } = useSelect( ( select ) => {
+		const { getEntityRecord, __experimentalGetCurrentGlobalStylesId } =
+			select( coreStore );
+
+		const globalStylesId = __experimentalGetCurrentGlobalStylesId();
+		const globalStyles = globalStylesId
+			? getEntityRecord( 'root', 'globalStyles', globalStylesId )
+			: undefined;
+
+		return {
+			revisionsCount:
+				globalStyles?._links?.[ 'version-history' ]?.[ 0 ]?.count ?? 0,
+		};
+	}, [] );
+
+	const hasRevisions = revisionsCount >= 2;
+	const { modified } = revisions?.[ 0 ] || {};
```

Comment:
Why `{}`?

Reply:
At this point in the function, we don't know yet if `revisions` is loaded, so without a fallback `{}` I think the destructuring could fail. Here's a fairly artificial example from the console:

<img width="707" alt="image" src="https://github.com/WordPress/gutenberg/assets/14988353/f94e172e-fef8-4ff1-a8d6-775348f0a7bb">

Reply:
Oh, my bad for skim reading. Thanks!

Reply:
I guess `revisions?.[ 0 ]?.modified` might also work? No need to change though, just musing 😄

Reply:
> I guess revisions?.[ 0 ]?.modified might also work?

Yep, `const modified = revisions?.[ 0 ]?.modified` would also work. I always enjoy these little code style discussions, it's quite illuminating. For me, if I'd started out just wanting to grab the `modified` property, I probably would have used `revisions?.[ 0 ]?.modified`, but while hacking around I was destructuring a few different props, and then whittled it back down to just `modified` so kept the destructuring around, even though it isn't really needed.

Funny how these things hang around! I might swap it out when I do another pass 🙂